### DISCUSSION TITLE
Traffic light colors

### DISCUSSION THREAD

**Discussion by pcamp** [GDQuest/learn-gdscript#268] [General]
In the USA, traffic light colors are called red, yellow and green.  The yellow light is occasionally called amber, but I've never heard it called orange.

**Comment by NathanLovato** [comment]
Ping @johnnygossdev, who wrote this one I think. Or perhaps somebody else tweaked it. Johnny is from the UK so it might be different from the US. In France, where I'm from, it's called orange so it didn't catch my attention but I don't know what it's called in English.

**Comment by cogwoz** [comment]
In the UK we usually use red, amber, green. If I recall I think I chose orange in the moment because amber may be less well known for anyone who doesn't have English as a first language and it's basically orange.

I'm not at my computer, but it seems the color in the color palette is closer to yellow anyway, so changing to yellow seems like a good idea.

**Comment by NathanLovato** [comment]
Thanks, I just opened an issue from this: #271
